src/ctapipe/reco/impact.py

```diff
@@ -2,68 +2,73 @@
 """
 
 """
```

Comment:
Please add a docstring here

Reply:
Still missing a docstring. Note that this appears in the documentation, so it should at least say what this module is about (as simple  as:
```py
"""Implementation of the ImPACT reconstruction algorithm
"""
```